packages/sample/src/main.ts

```diff
@@ -2,18 +2,31 @@ import { Command } from "commander";
 import {
     createLedPattern,
     ExpansionHub,
-    openConnectedExpansionHubs,
     LedPatternStep,
+    openConnectedExpansionHubs,
     RevHub,
 } from "@rev-robotics/expansion-hub";
+import { runServo } from "./commands/servo.js";
 
 const program = new Command();
 
 program
     .version("1.0.0")
     .option("-l --list", "List connected devices")
-    .option("--led", "Start led pattern")
-    .parse(process.argv);
+    .option("--led", "Start led pattern");
+
+program
+    .command("servo <channel> <pulseWidth> [frameWidth]")
+    .description("Run a servo with pulse width and optional frame width")
+    .action(async (channel, pulseWidth, frameWidth) => {
+        let channelValue = Number(channel);
+        let pulseWidthValue = Number(pulseWidth);
+        let frameWidthValue = frameWidth ? Number(frameWidth) : 4000;
+
+        await runServo(channelValue, pulseWidthValue, frameWidthValue);
+    });
```

Comment:
What happens if the user doesn't specify a channel or pulse width? Does `commander` automatically handle that somehow?

Reply:
If the user does not pass in a parameter defined with <foo>, we will see the error below. Optional parameters are defined as [bar].

`error: missing required argument 'channel'`

Reply:
In this case, channel and pulseWidth are required. The frameWidth is optional, and defaults to 4000.

Reply:
very cool!